syntax = "proto3";

package endpoints.notifications;

option java_multiple_files = true;
option java_outer_classname = "NotificationsProto";
option java_package = "com.google.endpoints.notification";

// Notifications service
service Notifications {
    rpc clientNotified(clientNotifiedRqst) returns (clientNotifiedResp);
}

// RPC Request and Response

message clientNotifiedRqst {
    string client_id = 1;
    string client_address = 2;
    string item_details = 3;
    string verbose = 4;
}

message clientNotifiedResp {
    string message = 1;
    string verbose = 2;
}